### PR TITLE
Add Gmail Desktop v2.20.0

### PR DESCRIPTION
**Pull request by weichenw**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/gmail-desktop.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask 'gmail-desktop' do
+  version '2.20.0'
+  sha256 '0cbde7446c1545000397b0b818eb0a7069d915438f5acea5315c844f3a74e5ca'
+
+  url "https://github.com/timche/gmail-desktop/releases/download/v2.20.0/gmail-desktop-#{version}-mac.dmg"
```
Comment (miccal) [inline Casks/gmail-desktop.rb:5]: Please change this to:
```
url "https://github.com/timche/gmail-desktop/releases/download/v#{version}/gmail-desktop-#{version}-mac.dmg"
```

Comment (weichenw) [reply Casks/gmail-desktop.rb]: Fixed the url. Thanks

File: Casks/gmail-desktop.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cask 'gmail-desktop' do
```
Comment (vitorgalvao) [inline Casks/gmail-desktop.rb:1]: Needs to be `timche-gmail-desktop`. Otherwise it looks erroneously official.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/gmail-desktop.rb (hunk 2 above)
Comment (weichenw) [reply Casks/gmail-desktop.rb]: You got a valid point there. I have renamed it to timche-gmail-desktop

**Pull request merged**